Apps/WebClient/src/ClientApp/src/views/faq.vue

```diff
@@ -249,7 +291,10 @@
                                 features and information. We prioritize our
                                 enhancements based on your feedback, and we
                                 encourage you to provide suggestions by
-                                e-mailing HealthGateway@gov.bc.ca
+                                e-mailing
+                                <a href="HealthGateway@gov.bc.ca"
```

Comment:
nit: with 'mailto' it will open the defaul mail program

https://css-tricks.com/snippets/html/mailto-links/